frontend/packages/ux-editor/src/testing/schemas/json/component/AddressComponent.schema.v1.json

```diff
@@ -1,85 +1,159 @@
 {
-  "$id": "https://altinncdn.no/schemas/json/component/AddressComponent.schema.v1.json",
+  "$id": "https://altinncdn.no/schemas/json/component/Address.schema.v1.json",
```

Comment:
Yes - apparently in RC2 of v4 the address component has changed name from `AddressComponent` to just `Address`. I guess this is to match the naming convention for the rest of the components. 
Will create a new issue to handle this change (I think there are several places that need to be updated), so part of that issue will also be to remove the old jsonschema.
Keeping them both for now to avoid breaking stuff 😊